CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.0.0
+- Creation of Package.swift file for the Swift package manager
```

Comment:
Do we need to include a note about the ios, tvos, macos version bumps here?

Reply:
Good point.

One thing that I should have made more clear in the PR - I have not actually restricted the package from being used in the lower OS versions (ie, 10, 11) - Just that I was unable to do testing on them due to the hardware limitations of the Mac. In theory it will work fine on those versions, so I have left them in (In the package.swift file, you can see the 3 minimum supported plaform versions)

However in saying that - this should be mentioned somewhere as a disclaimer. If we cannot test these out ourselves, then we need to make it clear that these old OS versions are not officially supported and are use at your own risk.

I'll add some stuff to the swift section of the Readme to cover this, cheers.